components/badge/badge.vue

```diff
@@ -121,6 +139,24 @@ export default {
       if (this.type === 'ai' && this.kind === 'count') {
         console.error('DtBadge error: type: \'ai\' with kind: \'count\' is an invalid combination.');
       }
```

Comment:
Might as well put this one into it's own function as well just to keep things clean.